generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/auth.interceptor.ts.ejs

```diff
@@ -33,7 +33,7 @@ export class AuthInterceptor implements HttpInterceptor {
     }
 
     intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-        if (!request || !request.url || (/^http/.test(request.url) && !(SERVER_API_URL && request.url.startsWith(SERVER_API_URL)))) {
+        if (!request || !request.url || (request.url.startsWith('http') && !(SERVER_API_URL && request.url.startsWith(SERVER_API_URL)))) {
```

Comment:
why did you change it?

Reply:
It reports an error due to typescript eslint [rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md) that's enabled by default as recommended configuration. I also prefer `startsWith` over regex in such scenarios as it's more readable.